drivers/ieee802154/ieee802154_rf2xx_iface.c

```diff
@@ -7,7 +7,11 @@
  */
```

Comment:
in you commit title add "for rf2xx driver"

Reply:
Done.